src/app/(proper_react)/redesign/(public)/LandingView.tsx

```diff
@@ -22,11 +22,13 @@ import PCMagLogo from "./social-proof-images/pcmag.svg";
 import TechCruchLogo from "./social-proof-images/techcrunch.svg";
 import { TelemetryLink } from "./TelemetryLink";
 import { HeresHowWeHelp } from "./HeresHowWeHelp";
+import { ScanLimit } from "./ScanLimit";
 
 export type Props = {
   eligibleForPremium: boolean;
   l10n: ExtendedReactLocalization;
   countryCode: string;
+  scanLimit: boolean;
```

Comment:
Same nit as in the other PR: `scanLimitReached` might be more descriptive (I'd expect `scanLimit` to be a number).